README.md

```diff
@@ -1,5 +1,59 @@
 # arxiv-search
 
+## Development quickstart
+
+### Alternatively: Running Elasticsearch with Kibana
+**TODO::** docker-compose version
+TODO: why is this the best
+```bash
+docker-compose up
+```
+Kibana will be available at http://127.0.0.1:5601/
```

Comment:
Consider including the commands to build the test index within Kibana.

Reply:
Will take another pass for documentation update